code/datums/spells/conjure.dm

```diff
@@ -15,7 +15,12 @@
 
 	var/cast_sound = 'sound/items/welder.ogg'
 
-/obj/effect/proc_holder/spell/aoe_turf/conjure/cast(list/targets,mob/living/user = usr)
+/obj/effect/proc_holder/spell/aoe/conjure/create_new_targeting()
+	var/datum/spell_targeting/aoe/turf/targeting = new()
+	targeting.range = aoe_range
+	return targeting
+
+/obj/effect/proc_holder/spell/aoe/conjure/cast(list/targets,mob/living/user = usr)
```

Comment:
```suggestion
/obj/effect/proc_holder/spell/aoe/conjure/cast(list/targets, mob/living/user = usr)
```

I also hate `usr` being in there... but cleaning that up would require a refactor and you'd have to check the usages. Not worth it in this PR